templates/guides/import-script.md

```diff
@@ -0,0 +1,115 @@
+Terraform version 1.9.7 is installed and meets the minimum required version.
+Importing deployments for organization cm23qgwp001ap01qm0o3er493
+Importing Deployments: [cf23qgwp001ag01qf0o8er413]
+Successfully handled resource deployment
+Successfully wrote import configuration to import.tf
+generated.tf does not exist, no need to delete
+terraform.tfstate does not exist, no need to delete
+astro_deployment.deployment_cf23qgwp001ag01qf0o8er413: Preparing import... [id=cf23qgwp001ag01qf0o8er413]
+astro_deployment.deployment_cf23qgwp001ag01qf0o8er413: Refreshing state... [id=cf23qgwp001ag01qf0o8er413]
+
+Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols:
+  + create
+  ~ update in-place
+
+Terraform will perform the following actions:
+
+...
+
+Plan: 1 to import, 6 to add, 1 to change, 0 to destroy.
+
+generated.tf does not exist. Creating new file with deployment information.
+Generated import for astro_deployment.deployment_cf23qgwp001ag01qf0o8er413
+Successfully updated generated.tf with deployment information.
+Import process completed successfully. The 'generated.tf' file now includes all resources, including deployments.
+Import process completed. Summary:
+Resource deployment processed successfully
+```
+
+## Step 3: Review output
+The script will generate two main files:
+- `import.tf`: Contains the Terraform import blocks for the specified resources.
+- `generated.tf`: Contains the Terraform resource configurations for the imported resources.
+The generated Terraform configurations may require some manual adjustment to match your specific requirements or to resolve any conflicts.
+
+## Step 4: Extract and organize resources
+The `generated.tf` file that is created by the import script will contain all of the specified resources in one file. It is recommended that you extract and modularize the resources so they are easily maintained and reusable:
+```
+terraform-astro-project/
+├── environments/
+│   ├── dev/
+│   │   ├── main.tf              # Root module for development
+│   │   ├── variables.tf         # Dev-specific variables
+│   │   ├── outputs.tf           # Dev-specific outputs
+│   │   └── dev.tfvars           # Variable values for development
+│   ├── prod/
+│   │   ├── main.tf              # Root module for production
+│   │   ├── variables.tf         # Prod-specific variables
+│   │   ├── outputs.tf           # Prod-specific outputs
+│   │   └── prod.tfvars          # Variable values for production
+├── modules/
+│   ├── astro/
+│   │   ├── main.tf              # Entry point for the module
+│   │   ├── workspace.tf         # Defines Astro workspaces
+│   │   ├── deployment.tf        # Defines Astro Deployments
+│   │   ├── users.tf             # Defines user roles and access
+│   │   ├── variables.tf         # Variables used in the astro module
+│   │   └── outputs.tf           # Outputs from the astro module
+└── cloud_provider.tf     
+```
```

Comment:
I would also include that this is just an example/suggestion file structure - each org is going to structure their repo differently